cpp/src/Slice/Grammar.y

```diff
@@ -16,18 +76,17 @@
 #   pragma warning(disable:4065)
 // warning C4244: '=': conversion from 'int' to 'yytype_int16', possible loss of data
 #   pragma warning(disable:4244)
+// warning C4127: conditional expression is constant
+#   pragma warning(disable:4127)
 #endif
 
-//
 // Avoid old style cast warnings in generated grammar
-//
 #ifdef __GNUC__
 #  pragma GCC diagnostic ignored "-Wold-style-cast"
+#  pragma GCC diagnostic ignored "-Wunused-label"
```

Comment:
I guess we need this warning for the generated Bison code?
It'd be good to update the comment on the enclosing 'ifdef' though.